### PR TITLE
Fix tests for edit profile #1675

### PR DESCRIPTION
**Pull request by OliaHrytsak**
Added tests to EditProfile component

image

### CODE REVIEW

**Review by mhevyk — APPROVED** [review]
Great work!

**Review by asya982 — APPROVED** [review]

File: tests/unit/pages/edit-profile/EditProfile.spec.jsx [hunk 1]
```diff
@@ -1,14 +1,108 @@
...
+
+    const menuTabs = await screen.findAllByRole('listitem')
+    expect(menuTabs).toHaveLength(3)
+  })
+
+  it('should render Profile Container after click on Profile menu button', async () => {
+    const profileMenuTab = await screen.findByRole('button', {
+      name: 'editTutor.main.profile'
+    })
+    act(() => {
```
Comment (Mav-Ivan) [inline tests/unit/pages/edit-profile/EditProfile.spec.jsx:74]: Do we need wrap by act here ?

Comment (OliaHrytsak) [reply tests/unit/pages/edit-profile/EditProfile.spec.jsx]: Really works without act. thank you!
Add this changes to request

File: tests/unit/pages/edit-profile/EditProfile.spec.jsx [hunk 2]
```diff
@@ -1,14 +1,102 @@
...
+const mockState = {
+  appMain: { userId: userId, userRole: userRole }
+}
+
+const userMock = 'UserProfileMock'
+
+vi.mock(
+  '~/containers/edit-profile/profile-general-tab/ProfileGeneralTab',
+  () => ({
+    __esModule: true,
```
Comment (iva-stasia) [inline tests/unit/pages/edit-profile/EditProfile.spec.jsx:19]: Please note that we don't need to use `__esModule: true` in the current Vitest version.

**Review by iva-stasia — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**